datumaro/components/operations.py

```diff
@@ -279,7 +279,12 @@ def _merge_videos(item_a: DatasetItem, item_b: DatasetItem) -> Video:
         media = None
 
         if isinstance(item_a.media, Video) and isinstance(item_b.media, Video):
-            raise VideoMergeError(item_a.id)
+            if item_a.media.path is not item_b.media.path or \
+                    item_a.media.length() is not item_b.media.length() or \
+                    item_a.media.frame_size() is not item_b.media.frame_size():
```

Comment:
I think, for video we should check path, start, end frame and step instead. Using length and frame size is not reliable and doesn't really give us any information. Probably, we should make `start`, `end` frame and `step` read-only properties in the `Video` class. Please do this in a separate PR.